src/AddressChecker.js

```diff
@@ -45,15 +45,21 @@ class AddressChecker extends Component {
     });
   };
   handleChangeSegwit = e => {
-    this.setState({ segwit: !this.state.segwit });
+    this.setState({ 
+      segwit: !this.state.segwit,
+      path: `${(this.state.segwit? 44 : 49)}'/${this.state.coin}'/0'/0/0` 
```

Comment:
note 44 : 49 seems the wrong way round, but since this refers to the old state before the toggle it is right